pkg/state/state_ipoib_network_test.go

```diff
@@ -150,24 +153,23 @@ var _ = Describe("IPoIBNetwork Network state rendering tests", func() {
 			Expect(errors.IsNotFound(err)).To(BeTrue())
 			err = client.Get(context.Background(), types.NamespacedName{Namespace: testNamespace, Name: name}, nad)
 			Expect(err).NotTo(HaveOccurred())
-			expectedNad := getExpectedIPoIBNAD(name, "{}")
+			expectedNad := getExpectedIPoIBNAD(name, testMaster, "{}")
 			Expect(nad.Spec).To(BeEquivalentTo(expectedNad.Spec))
 			Expect(nad.Name).To(Equal(name))
 			Expect(nad.Namespace).To(Equal(testNamespace))
 		})
 	})
 })
 
-func getExpectedIPoIBNAD(testName, ipam string) *netattdefv1.NetworkAttachmentDefinition {
+func getExpectedIPoIBNAD(testName, testMaster, ipam string) *netattdefv1.NetworkAttachmentDefinition {
 	nad := &netattdefv1.NetworkAttachmentDefinition{}
-	cfg := fmt.Sprintf("{ \"cniVersion\":\"0.3.1\", \"name\":%q, \"type\":\"ipoib\", "+
-		"\"master\": %q, \"ipam\":%s }",
+	cfg := fmt.Sprintf(`{ "cniVersion":"0.3.1", "name":%q, "type":"ipoib", "master": %q, "ipam":%s }`,
 		testName, testMaster, ipam)
 	nad.Spec.Config = cfg
 	return nad
 }
 
-func getIPoIBNetwork() *mellanoxv1alpha1.IPoIBNetwork {
+func getIPoIBNetwork(testNamespace, testMaster string) *mellanoxv1alpha1.IPoIBNetwork {
```

Comment:
Can fix the linter warning? If we want to keep the arg it is ok to add a linter exception here.